packages/usa-combo-box/src/test/test-patterns/test-usa-combo-box.twig

```diff
@@ -1,26 +1,30 @@
 <div class="padding-2">
   <form class="usa-form padding-x-2 padding-bottom-2 padding-top-1px bg-base-lightest radius-md">
-    {% include '../usa-combo-box/src/usa-combo-box.twig' with {
-      'label': 'Default combo box in a usa-form'
+    {% include '@components/usa-combo-box/src/usa-combo-box.twig' with {
+      'label': 'Default combo box in a usa-form',
+      'name': 'fruit',
+      'id': 'fruit'
     } %}
 
-    {% include '../usa-combo-box/src/usa-combo-box.twig' with {
+    {% include '@components/usa-combo-box/src/usa-combo-box.twig' with {
       'modifier': 'width-tablet maxw-none',
       'label': 'Override combo box with utilities',
       'name': 'more-fruit',
       'id': 'more-fruit'
     } %}
   </form>
 
-  {% include '../usa-combo-box/src/usa-combo-box.twig' with {
+  {% include '@components/usa-combo-box/src/usa-combo-box.twig' with {
     'modifier': 'width-tablet maxw-none',
     'label': 'Not in a usa-form element',
-    'name': 'even-more-fruit'
+    'name': 'even-more-fruit',
+    'id': 'even-more-fruit'
   } %}
 
-  {% include '../usa-combo-box/src/usa-combo-box.twig' with {
+  {% include '@components/usa-combo-box/src/usa-combo-box.twig' with {
     'modifier': 'width-full maxw-none',
     'label': 'Not in a usa-form element, custom width utilities',
-    'name': 'even-more-more-fruit'
+    'name': 'even-more-more-fruit',
+    'id': 'even-more-more-fruit'
   } %}
 </div>
```

Comment:
@mejiaj I am not sure how this is supposed to render but this is the test result:
![image](https://user-images.githubusercontent.com/93996430/162757011-5a7dab23-f5aa-4952-a4cb-3a4727585cfe.png)
Is the expectation that the gray background extends to include the wider box content?

Reply:
@amyleadem, yes I think so. That was to illustrate utility overrides.

Reply:
Added maxw-none class to the form to match the class on the select wrapper in commit [343e0b1](https://github.com/uswds/uswds/pull/4611/commits/343e0b1d705fa07e2d7f561e5ba2db85872920e2). If this is breaking the test at all, please let me know.